### PR TITLE
FE-1670 Cypress maintenance phase

### PR DESCRIPTION
**Pull request by DlgSHi**
First part of maintenance fixes for bd specs.

### CODE REVIEW

File: cypress/features/regression/message.feature [hunk 1]
```diff
@@ -65,14 +64,13 @@ Feature: Message component
 
   @positive
   Scenario: Enable on dismiss state for a Message component
```
Comment (marcinstachowicz) [inline cypress/features/regression/message.feature]: I think this should be split to two scenarios

File: cypress/features/regression/pod.feature [hunk 2]
```diff
@@ -6,13 +6,12 @@ Feature: Pod component
 
   @positive
   Scenario: Enable border checkbox for a Pod component
-    When I check border checkbox
+    # When I check border checkbox
```
Comment (marcinstachowicz) [inline cypress/features/regression/pod.feature]: When I uncheck
And I check
:)

File: cypress/features/regression/preview.feature [hunk 3]
```diff
@@ -6,13 +6,12 @@ Feature: Preview component
 
   @positive
   Scenario: Enable loading checkbox for a Preview component
-    When I check loading checkbox
+    # When I check loading checkbox
```
Comment (marcinstachowicz) [inline cypress/features/regression/preview.feature]: When I uncheck
And I check

File: cypress/features/regression/row.feature [hunk 4]
```diff
@@ -6,13 +6,12 @@ Feature: Row component
 
   @positive
   Scenario: I check columnDivide
-    When I check columnDivide checkbox
+    # When I check columnDivide checkbox
```
Comment (marcinstachowicz) [inline cypress/features/regression/row.feature]: When I uncheck
Then I check

File: cypress/features/regression/row.feature [hunk 5]
```diff
@@ -6,13 +6,12 @@ Feature: Row component
 
   @positive
   Scenario: I check columnDivide
```
Comment (marcinstachowicz) [inline cypress/features/regression/row.feature]: I enable columnDivide

File: cypress/features/regression/row.feature [hunk 6]
```diff
@@ -6,13 +6,12 @@ Feature: Row component
...
-    When I check columnDivide checkbox
+    # When I check columnDivide checkbox
     Then columnDivide is set
 
   @positive
   Scenario: I uncheck columnDivide
```
Comment (marcinstachowicz) [inline cypress/features/regression/row.feature]: I disable

File: cypress/features/regression/settingsRow.feature [hunk 7]
```diff
@@ -48,11 +48,10 @@ Feature: Settings Row component
 
   @positive
   Scenario: Enable divider checkbox for a Settings Row component
-    When I check divider checkbox
+    # When I check divider checkbox
```
Comment (marcinstachowicz) [inline cypress/features/regression/settingsRow.feature]: When I uncheck
Then I check

File: cypress/features/regression/switch.feature [hunk 8]
```diff
@@ -67,13 +67,12 @@ Feature: Switch component
 
   @positive
   Scenario: Enable reverse property
-    When I check reverse checkbox
+    # When I check reverse checkbox
```
Comment (marcinstachowicz) [inline cypress/features/regression/switch.feature]: same here

File: cypress/support/step-definitions/date-input-steps.js [hunk 9]
```diff
@@ -6,6 +7,8 @@ import { labelPreview } from '../../locators';
 const TEXT_ALIGN = 'text-align';
 const DAY = 'DayPicker-Day--';
 const TODAY_DAY = Cypress.moment().format('ddd MMM D, YYYY');
+const YESTERDAY_DAY = moment().subtract(1, 'days').format('ddd MMM D, YYYY');
```
Comment (marcinstachowicz) [inline cypress/support/step-definitions/date-input-steps.js]: Cypress.moment()

File: cypress/support/step-definitions/multi-action-button-steps.js [hunk 10]
```diff
@@ -23,20 +22,34 @@ Then('Multi Action Button state is not disabled', () => {
 });
 
 Then('Multi Action Button as on preview is {string}', (asProperty) => {
-  multiActionButtonPreview()
-    .children()
-    .should('have.class', `${MULTI_ACTION_BUTTON_AS_PROPERTY}${asProperty}`);
+  switch (asProperty) {
+    case 'primary':
+      multiActionButtonPreview().children()
+        .should('have.css', 'background-color', 'rgb(30, 73, 159)');
```
Comment (marcinstachowicz) [inline cypress/support/step-definitions/multi-action-button-steps.js]: is it possible to add other css condition?

File: cypress/features/regression/message.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/message.feature]: fixed

File: cypress/features/regression/pod.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/features/regression/pod.feature]: fixed

File: cypress/features/regression/preview.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/regression/preview.feature]: fixed

File: cypress/features/regression/row.feature (hunk 4 above)
Comment (DlgSHi) [reply cypress/features/regression/row.feature]: fixed

File: cypress/features/regression/row.feature (hunk 5 above)
Comment (DlgSHi) [reply cypress/features/regression/row.feature]: fixed

File: cypress/features/regression/row.feature (hunk 6 above)
Comment (DlgSHi) [reply cypress/features/regression/row.feature]: fixed

File: cypress/features/regression/settingsRow.feature (hunk 7 above)
Comment (DlgSHi) [reply cypress/features/regression/settingsRow.feature]: fixed

File: cypress/features/regression/switch.feature (hunk 8 above)
Comment (DlgSHi) [reply cypress/features/regression/switch.feature]: fixed

File: cypress/support/step-definitions/date-input-steps.js (hunk 9 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/date-input-steps.js]: fixed

File: cypress/support/step-definitions/multi-action-button-steps.js (hunk 10 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/multi-action-button-steps.js]: fixed

File: cypress/features/regression/message.feature [hunk 11]
```diff
@@ -70,6 +70,13 @@ Feature: Message component
       And I click dismiss icon
       And click action was called in Actions Tab
 
+  @positive
+  Scenario: Verify the click function for a Message component
+    # When I check onDismiss checkbox
+    Then clear all actions in Actions Tab
+      And I click dismiss icon
+      And click action was called in Actions Tab
```
Comment (marcinstachowicz) [inline cypress/features/regression/message.feature]: When Then

File: cypress/support/step-definitions/date-input-steps.js [hunk 12]
```diff
@@ -1,4 +1,4 @@
-import moment from 'moment';
+// import moment from 'moment';
```
Comment (marcinstachowicz) [inline cypress/support/step-definitions/date-input-steps.js]: remove unused import

File: cypress/features/regression/message.feature (hunk 11 above)
Comment (DlgSHi) [reply cypress/features/regression/message.feature]: fixed

File: cypress/support/step-definitions/date-input-steps.js (hunk 12 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/date-input-steps.js]: removed

**Pull request merged**